### ISSUE TITLE
Approximate Power-of-2(AP2) 구현

### ISSUE THREAD

**Issue by ssaru** [DeepBaksuVision/Binarized_Neural_Network#10]

**Comment by ssaru** [comment]
1. signum function은 PyTorch의 torch.sign 사용하기
https://pytorch.org/docs/stable/torch.html#torch.sign

2. log function 또한 PyTorch에서 log함수 사용
https://pytorch.org/docs/stable/torch.html#torch.log

3. Absolute operation 적용 빠짐
3페이지 오른쪽 상단 shifted based batch normalization파트에서 AP2 공식 자세히 볼 것
https://arxiv.org/pdf/1602.02830.pdf
